src/api/tasks/TaskCollaborators.js

```diff
@@ -66,6 +70,43 @@ class TaskCollaborators extends TasksBase {
         });
     }
 
+    createTaskCollaboratorsforGroup({
+        errorCallback,
+        file,
+        successCallback,
+        task,
+        group,
+    }: {
+        errorCallback: ElementsErrorCallback,
+        file: BoxItem,
+        group: { id: string },
+        successCallback: Function,
+        task: { id: string },
+    }): void {
+        this.errorCode = ERROR_CODE_CREATE_TASK_COLLABORATOR;
```

Comment:
Should this have its own error code, since it's a different endpoint?

Reply:
Yes! That will be coming up in a follow up PR